contrib/cirrus/rootless_test.sh

```diff
@@ -2,11 +2,13 @@
 
 set -e
 
+echo "$(date --rfc-3339=seconds) $(basename $0) started with '$*'"
+
```

Comment:
I think this is the second (third?) time I've seen this line.  I realize it's a one-liner, but do you think this might be useful to have as a function in `lib.sh`.  Really it would just save a bit of typing and guarantee fewer typos (by me!) :grinning: 

I'm fine either way, it's up to you.

Reply:
Two instances. I considered refactoring when I added the second, but IMHO it's not worth the effort just yet. Possibly some day with a different kind of cleanup. Again, the ultra-high-priority purpose of this PR is to get rid of the nasty `cgroup.freeze` flakes which are wasting a lot of everyone's time. Everything else is just some nice-to-haves that I tacked on.